### DISCUSSION TITLE
理工饒舌歌手Robin Gan

### DISCUSSION THREAD

**Discussion by PaulSu0905** [1101-GS4719-C/Python-Community-GS4719-C#1286] [General]
不知道各位平常是否會聽饒舌歌呢？
最近發現一個很酷的饒舌歌手Robin Gan
一位就讀美國華盛頓大學的華裔學生
他在自己的歌詞中寫了許多理工男的心聲
甚至將Java,數學,三角函數等等都寫入歌詞中
聽完他的歌之後，真的覺得自己還不夠認真，沒有將所學發揮的淋漓盡致，也佩服他勇於表達自己的理工男精神，抨擊那些看不起理工男的人
以下附上他的YouTube 頻道：https://www.youtube.com/@RobinGan/featured
想知道大家聽完他的歌之後有哪些有趣的看法？

**Comment by poweichen00** [comment]
我覺得他所寫的歌蠻有趣的，把理工男日常寫入歌，這是目前網路上沒有出現過，在作品上更能凸顯個人特色。
